naga/src/ir/mod.rs

```diff
@@ -2141,6 +2178,8 @@ pub enum Statement {
         /// The specific operation we're performing on `query`.
         fun: RayQueryFunction,
     },
+    /// A mesh shader intrinsic
```

Comment:
nitpick: To be consistent with other comments here, let's terminate this sentence with a period.


```suggestion
    /// A mesh shader intrinsic.
```